Source/Scene/Cesium3DTilesetMetadata.js

```diff
@@ -15,55 +15,62 @@ import TilesetMetadata from "./TilesetMetadata.js";
  * </p>
  *
  * @param {Object} options Object with the following properties:
- * @param {String} options.extension The extension JSON object.
+ * @param {Object} options.tilesetJson The tileset JSON object.
```

Comment:
I would stick with the `metadataJson` instead. While a tileset JSON may be a valid object with metadata, if you use the `3DTILES_metadata` extension, it's not true.

Also please document this parameter more to explain the two cases that are valid, thanks!